server/plugin_test.go

```diff
@@ -135,13 +135,13 @@ func TestConfigCommand(t *testing.T) {
 			})
 
 			roles := "system_user"
-			if val.isAdmin {
+			if test.isAdmin {
 				roles += " system_admin"
 			}
 
 			mockAPI.On("GetUser", mock.AnythingOfType("string")).Return(&model.User{Id: "123", Roles: roles}, nil)
```

Comment:
You can create a function like `GetMockUser(isAdmin bool)` and call that function here.